app/services/balance_transactions/updater.rb

```diff
@@ -0,0 +1,32 @@
+class BalanceTransactions::Updater
+  Args = Struct.new(:date, :comment, :amount, :transaction_id)
+
+  def initialize(args)
+    @args = Args.new(args[:date], args[:comment], args[:amount], args[:transaction_id])
+  end
+
+  def call
+    ActiveRecord::Base.transaction do
+      update_balance_transaction
+      update_transaction_data
+    end
+    @_transaction
```

Comment:
@sinpai It is a Updator service, I think need return `true` or `false`, how all `update` methods.